chandra_aca/tests/test_all.py

```diff
@@ -71,15 +77,52 @@ def test_aca_targ_transforms():
 
 
 def test_t_ccd_warm_limit():
-    out = t_ccd_warm_limit([9.8] * 6, date='2015:001', min_n_acq=(2, 8e-3))
-    assert np.allclose(out[0], -13.3341, atol=0.01)
+    out = t_ccd_warm_limit([10.4] * 6, date='2015:001', min_n_acq=(2, 8e-3))
+    assert np.allclose(out[0], -15.2449, atol=0.01)
     assert np.allclose(out[1], 0.008, atol=0.0001)
 
-    out = t_ccd_warm_limit([9.8] * 6, date='2015:001', min_n_acq=5.0)
-    assert np.allclose(out[0], -13.2155, atol=0.01)
+    out = t_ccd_warm_limit([10.4] * 6, date='2015:001', min_n_acq=5.0)
+    assert np.allclose(out[0], -15.143, atol=0.01)
```

Comment:
Process fail.  Tests do not actually pass for 0.6 with the probability model update but it got installed.  This will be a candidate for Travis testing when we revive skare-conda..